feature/google/src/main/java/com/revenuecat/purchases/google/GoogleStoreProduct.kt

```diff
@@ -1,8 +1,12 @@
-package com.revenuecat.purchases.models
+package com.revenuecat.purchases.google
```

Comment:
Is this data class meant to be accessible by devs? If so, moving it to the `google` module will hide it, it's not directly exposed by just importing our SDK.

Reply:
I don't _think_ its supposed to be accessible? But I don't actually know 🤔 

 I was moving it because the `AmazonStoreProduct` is in the `amazon` module. I forgot I did this and should have called out this change in the description (I will go update that).
 
@vegaro Can devs still access it by importing the  `google` module some how?

Reply:
Mentioned it before but the `feature/google` module is not supposed to be accesible by devs. `feature/amazon` will be though, so it's different between them.

I believe the plan was for this to be accesible in case devs needed access to the underlying data, though it shouldn't be needed to build a paywall

Reply:
@tonidero Oh, I wasn't aware of that! My bad 😱 

But... if the underlying data isn't needed for a paywall and won't be used in most cases, should the store specific models all live under `feature/google`? 🤔

Reply:
> should the store specific models all live under feature/google?

Well, if we don't allow devs to get that data, there might be some users that complain since that's something that devs can currently do with BC4. That said, I'm not aware of any use cases that would require it other than devs being able to handle google data on their own, and not sure how important that is...

Reply:
agh, i remember we had a conversation a while back about it being important for devs to access ProductDetails if they want to.... though perhaps we could just require they import that module if they really want it?

Reply:
I think it would be very difficult for devs to access the `feature/google` module directly since it won't be accessible using the published SDK afaik. So if we think it's important for devs to access it, it should be part of the public API in the `public`/`purchases` modules.